cypress/e2e/shared/explorerVisualizations.test.ts

```diff
@@ -415,6 +415,8 @@ describe('visualizations', () => {
     })
 
     it('can view table data & scroll to bottom', () => {
+      // to fit all the elements in screen we force a view port size
+      cy.viewport(1200, 1000)
```

Comment:
just a word of caution, we had issues in the past when tests were modifying the viewport in the middle of the test. Would be preferred if we could avoid this with another solution

Reply:
@genehynson is it possible to set the viewport globally? it seems to be pretty small for our app

Reply:
totes! https://docs.cypress.io/api/commands/viewport#Defaults Just toss it into the `cypress.json` file in the UI repo

Reply:
✅